lib_xud/doc/rst/index.rst

```diff
@@ -1,11 +1,13 @@
 
-.. include:: ../../../README.rst
-
+#############################
```

Comment:
Is the remove of the Read Me file from this documentation intentional?

Reply:
This was breaking the documentation build (due to heading mismatches) also the information is duplicated.

Reply:
Okay.